dds/DCPS/RTPS/RtpsDiscovery.cpp

```diff
@@ -54,9 +54,13 @@ RtpsDiscoveryConfig::RtpsDiscoveryConfig()
   , dx_(2)
   , ttl_(1)
   , sedp_multicast_(true)
-  , default_multicast_group_("239.255.0.1") /*RTPS v2.1 9.6.1.4.1*/
+  , sedp_local_address_(static_cast<u_short>(0), "0.0.0.0")
```

Comment:
There's a lot of these `static_cast`s here.  Consider one of these options:

- named constant `PORT_ZERO` (or similar) at file scope (anonymous namespace)
- functional cast expression `u_short(0)`
- helper function `ACE_INET_Addr fn(const char*)`